### ISSUE TITLE
task2

### ISSUE THREAD

**Issue by tuchang** [tuchang/PyTorch-datawhale-201908#2]
1. numpy和pytorch实现梯度下降法
2. 设定初始值
3. 求取梯度
4. 在梯度方向上进行参数的更新
5. numpy和pytorch实现线性回归
6. pytorch实现一个简单的神经网络

张量
bilibili 吴恩达 gradient descent

PyTorch中，所有神经网络的核心是 autograd 包
autograd 包为张量上的所有操作提供了自动求导机制。它是一个在运行时定义（define-by-run）的框架，这意味着反向传播是根据代码如何运行来决定的，并且每次迭代可以是不同的.
反向传播算法
雅可比矩阵

Octave
